malaria-app-android/src/main/res/xml/appwidget_info.xml

```diff
@@ -2,7 +2,7 @@
 <appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
     android:minWidth="200dp"
     android:minHeight="40dp"
-    android:updatePeriodMillis="100"
+    android:updatePeriodMillis="86400000"
```

Comment:
Can we not hardcode this and instead put in integer.xml? with an appropriate desc